src/main/java/org/entando/entando/plugins/jpmail/ent/system/services/model/EmailSenderDto.java

```diff
@@ -16,15 +16,20 @@
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotBlank;
 import java.util.Objects;
+import javax.validation.constraints.Pattern;
 
 public class EmailSenderDto {
 
     public EmailSenderDto() {
     }
 
     public EmailSenderDto(@NotBlank(message = "error.emailSender.code.notBlank") String code,
-                          @NotBlank(message = "error.emailSender.email.notBlank")
-                          @Email(message = "error.emailSender.email.notValid") String email) {
+            @NotBlank(message = "error.emailSender.email.notBlank")
+            @Email(message = "error.emailSender.email.notValid")
```

Comment:
I dislike this approach (using multiple annotations to validate parts of the same field can be confusing). What about replacing @Email with a single regex?